src/NetscriptFunctions/Singularity.ts

```diff
@@ -736,48 +738,48 @@ export function NetscriptSingularity(): InternalAPI<ISingularity> {
     applyToCompany: (ctx) => (_companyName, _field) => {
       helpers.checkSingularityAccess(ctx);
       const companyName = getEnumHelper("CompanyName").nsGetMember(ctx, _companyName);
-      const field = helpers.string(ctx, "field", _field);
+      const field = getEnumHelper("JobField").nsGetMember(ctx, _field, "field");
```

Comment:
`singularity.applyToCompany` now throws an exception on an invalid job field since it's using the enum validator. This is different behaviour to before as an invalid field would just log an error, returning false. Happy to revert if this is too breaking.